src/ludamus/templates/panel/facilitators.html

```
@@ -51,26 +101,69 @@
-                                <a href="{% url 'panel:facilitator-edit' slug=current_event.slug facilitator_slug=facilitator.slug %}"
-                                   class="text-primary hover:text-primary/80 inline-flex items-center">
-                                    {% icon "pencil-square" class="w-4 h-4 mr-1" %}
-                                    {% translate "Edit" %}
-                                </a>
+                                <div class="inline-flex items-center gap-3 justify-end">
+                                    {% if facilitator.accreditation_type != "guest" %}
+                                        <form method="post"
+                                              action="{% url 'panel:facilitator-mark-guest' slug=current_event.slug facilitator_slug=facilitator.slug %}">
+                                            {% csrf_token %}
+                                            <input type="hidden" name="next" value="{{ request.get_full_path }}">
+                                            <button type="submit"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)